src/zui/components/ZUIAvatarGroup/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import { FC } from 'react';
+
+import ZUIAvatar from '../ZUIAvatar';
+import { ZUISize } from '../types';
+
+type ZUIAvatarGroupProps = {
+  /**
+   * List of the people you want to display as avatars.
+   */
+  avatars: {
+    firstName: string;
+    id: number;
+    lastName: string;
+  }[];
+
+  /**
+   * Maximum number of avatars shown.
+   */
+  max?: number;
+
+  /**
+   * The size of the avatars. Defaults to 'medium'.
+   */
+  size?: ZUISize;
+
+  /**
+   * The shape of the avatars. Defaults to 'circular.
+   */
+  variant?: 'circular' | 'square';
+};
+
+const ZUIAvatarGroup: FC<ZUIAvatarGroupProps> = ({
+  avatars,
+  max,
+  size = 'medium',
+  variant = 'circular',
+}) => {
+  const theme = useTheme();
+
+  let avatarSize = '2rem';
+  let fontSize = '0.875rem';
+  if (size == 'small') {
+    avatarSize = '1.5rem';
+    fontSize = '0.625rem';
+  } else if (size == 'large') {
+    avatarSize = '3rem';
+    fontSize = '1rem';
+  }
+
+  const showOverflowNumber = max && max < avatars.length;
```

Comment:
This expression is causing a bug. When `max == 0`, the first operand is falsy, and the way that the javascript `&&` operator works, when the first operand is falsy, the expression evaluates _to the value of that operand_ and _not to `false`_. So when `max == 0`, that means that `showOverflowNumber == 0`.

Reply:
Fixed in 7fc52464ca1c9a89e07632ac919d95378bd21411.